### ISSUE TITLE
resolveViewModel should report file name if it can't find a class

### ISSUE THREAD

**Issue by timfish** [MeirionHughes/aurelia-template-lint#145]
I'm using `gulp-aurelia-template-lint@0.9.1`

But it's tripping up due to the following code in `binding.ts`:

```typescript
    let first = classes[0];
    let context = new ASTContext();

    context.name = first.name.getText();
```

I can't for the life of me work out which file its having problems with. I'm guessing I have an empty file somewhere? Would be helpful to have a file path!

```shell
C:\Users\tim\Documents\My-Project\node_modules\aurelia-template-lint\dist\rules\binding.js:213
        context.name = first.name.getText();
                            ^

TypeError: Cannot read property 'name' of undefined
    at BindingRule.resolveViewModel (C:\Users\tim\Documents\My-Project\node_modules\aurelia-template-lint\dist\rules\binding.js:213:29)
    at BindingRule.init (C:\Users\tim\Documents\My-Project\node_modules\aurelia-template-lint\dist\rules\binding.js:23:34)
  ... snip

```

**Comment by MeirionHughes** [comment]
I'll try and triage asap

**Comment by MeirionHughes** [comment]
Try 0.9.20; it should stop complaining that it cannot find the view model; as of 0.9.20 the default config is not to complain. As for your case, it seems its finding the source file but cannot find any classes in it.
